### DISCUSSION TITLE
Framework - Create record WITHOUT validations doesn't trigger realtime events

### DISCUSSION THREAD

**Discussion by jimt** [pocketbase/pocketbase#1534] [Q&A] [answered]
I am experimenting with using PocketBase as a framework and was creating lots of records in my go code, but I was frustrated my Javascript client subscription to '*' on the collection was never seeing an event. When I switched from WITHOUT validations to WITH validations, the events started flowing.  Is that an intentional difference?

**Comment by ganigeorgiev** [comment, marked as answer]
The realtime events are binded to the `app.OnModel*` hooks, which are triggered in both cases - when saving a record using the `app.Dao` or the `RecordUpsert` form.

For the wildcard topic (aka. `*`) is required the client to satisfy the "List API" rule of your subscribed collection.

If that's not the case, please provide a code sample how you are saving the record and what version of PocketBase you are using.

**Reply by jimt** [reply]
Thank you for the fast response! :clap: 

Yes, while prototyping, I had made both the List/Search and View API rules empty.

Thank you for the confirmation.  I thought the events started flowing when I changed from `a.Dao().SaveRecord()` to `form.LoadData()...form.Submit()` but perhaps it was something else.  I will have to investigate further.  Sorry for the noise.

Thanks for the interesting "framework" to explore.
